src/recordings.js

```diff
@@ -47,71 +55,41 @@ export function retrieveListOfSpeakers() {
-      // put the list item into the DOM
-      setupButton(individualSpeaker, recordingData)
-      recordingsList.appendChild(individualSpeaker.element)
-    }
-  }
-
-  function setupButton(createdSpeakerButton, recordingData) {
-    // Add appropriate text
-    createdSpeakerButton.slot['speaker-name'] = recordingData['speaker_name']
-    // TODO: this should be derived from the recording JSON
-    // TODO: as of 2020-06-04, it does not include this data :(
-    createdSpeakerButton.slot['speaker-dialect'] = recordingData['dialect']
-
-    // Setup audio
-    let audio = new Audio(recordingData.recording_url)
-    audio.preload = 'none'
-    createdSpeakerButton.element.addEventListener('click', () => {
-      audio.play()
-      displaySpeakerBioLink(recordingData)
-    })
-  }
-
-  // the function that creates a link for an individual speaker's bio to be clicked
-  function displaySpeakerBioLink(recordingData) {
-    // the URL to be placed into the DOM
-    let insertedURL = recordingData['speaker_bio_url']
-
-    // select for the area to place the speaker link
-    let container = document.querySelector('.speaker-link')
-
-    // variable (speaker's name) to be inserted into the DOM
-    let speakerName = recordingData['speaker_name']
-
-    // generate a new link and append it to the page if there isn't already one
-    if (container.childElementCount < 1) {
-      // clone our template so we can insert it into the DOM
-      let createdLink = SimpleTemplate.fromId('template:speaker-bio-link')
-
-      // set the speaker-link text with the name of the speaker
-      createdLink.slot['speaker-name'] = speakerName
-
-      // set the link URL
-      createdLink.element.href = insertedURL
-
-      // and place the node into the DOM
-      container.appendChild(createdLink.element)
-    } else {
-      // remove the node that was created:
-      container.removeChild(container.childNodes[1]) // may need to extract the inner parameter based on Eddie's feedback...
-
-      // create a new node for the new speaker name
-      let newSpeakerNode = SimpleTemplate.fromId('template:speaker-bio-link')
-      // ...and place the newly clicked speaker's name into it
-      newSpeakerNode.slot['speaker-name'] = speakerName
-
-      // get the URL again and reinsert into the newly created node
-      newSpeakerNode.element.href = insertedURL
-
-      // place said node into the DOM
-      container.appendChild(newSpeakerNode.element)
-    }
+        for (let recordingData of recordings) {          
+          // TODOkobe: if all is working/Eddie has approved, perhaps refactor into a function?
+          // create a option element
+          let listOption = document.createElement('option')
+          
+          // create a new textnode that is the speaker's name
+          let nodeSpeakerName = document.createTextNode(recordingData.speaker_name + ', ' + recordingData.dialect)
```

Comment:
I would use a `<template>` for the code that generates the `<option>` ¯\\\_(ツ)\_/¯

Reply:
Oof, this is the only one I left as ✨ outstanding✨ . I can squeeze it in pre-general meeting I think but if you think it can come after the fact, I can modify it later too. Out of curiosity, why a template over this ~~hamfisted version~~?

Reply:
The `<template>` is in the HTML, which is easier to localize, and easier to edit by, e.g., @aarppe. Left as is, to change this text we'd have to dig into the JavaScript code, when this particular matter is more about the _presentation_ of the speaker name/dialect. JavaScript should, in my mind, avoid presentational concerns as much as possible. Does that make sense?

Reply:
Yeah that makes sense! You want the JS to handle behaviour and not presentation – I gotchu 👌🏿